testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java

```diff
@@ -65,8 +65,19 @@ public Pattern[] get(boolean inContainer) {
                     // Podman with cgroupv2 on RHEL 9 intermittently spits out this message to no apparent effect on our tests
                     Pattern.compile(".*level=error msg=\"Cannot get exit code: died not found: unable to find event\".*"),
                     Pattern.compile(".*time=.*level=warning.*msg=.*S.gpg-agent.*since it is a socket.*"),
+                    // Podman -> registry network/comm issue?
+                    Pattern.compile(".*time=.*level=warning.*msg=.*Failed, retrying in.*pull&service=quay\\.io.*: net/http: TLS handshake timeout.*"),
                     // Testcontainers, depends on local setup. Not our test issue.
                     Pattern.compile(".*Please ignore if you don't have images in an authenticated registry.*"),
+                    // Common new Q versions
+                    Pattern.compile(".*io.quarkus.narayana.jta.runtime.graal.DisableLoggingFeature.*"),
+                    // Podman / Docker extension incompatibilities with Podman versions
+                    Pattern.compile(".*Database JDBC URL \\[undefined/unknown\\].*"),
+                    Pattern.compile(".*Database driver: undefined/unknown.*"),
+                    Pattern.compile(".*Autocommit mode: undefined/unknown.*"),
+                    Pattern.compile(".*Minimum pool size: undefined/unknown.*"),
+                    Pattern.compile(".*Isolation level: <unknown>.*"),
+                    Pattern.compile(".*Maximum pool size: undefined/unknown.*"),
```

Comment:
It feels like you are doing more things than advertised in this PR. Could you please update the PR title/description to reflect these changes as well?